tools/syz-trace2syz/parser/parser_test.go

```diff
@@ -0,0 +1,188 @@
+package parser
+
+import (
+	"github.com/google/syzkaller/prog"
+	_ "github.com/google/syzkaller/sys"
+	"testing"
+)
+
+var (
+	OS   = "linux"
+	Arch = "amd64"
+)
+
+func TestParseLoopBasic(t *testing.T) {
+	tests := []string{
+		`open() = 3` + "\n" +
+			`fstat() = 0`,
+		`open() = 0x73ffddabc` + "\n" +
+			`fstat() = 0`,
+		`open() = -1 ENOSPEC (something)` + "\n" +
+			`fstat() = 0`,
+		`open( ,  <unfinished ...>` + "\n" +
+			`<... open resumed>) = 3` + "\n" +
+			`fstat() = 0`,
+		`open( ,  <unfinished ...>` + "\n" +
+			`<... open resumed> , 2) = 3` + "\n" +
+			`fstat() = 0`,
+		`open( <unfinished ...>` + "\n" +
+			`<... open resumed>) = 3` + "\n" +
+			`fstat() = 0`,
+		`open( <unfinished ...>` + "\n" +
+			`<... open resumed>) = 0x44277ffff` + "\n" +
+			`fstat() = 0`,
+		`open( <unfinished ...>` + "\n" +
+			`<... open resumed>) = ?` + "\n" +
+			`fstat() = 0`,
+		`open( <unfinished ...>` + "\n" +
+			`<... open resumed>) = -1 FLAG (sdfjfjfjf)` + "\n" +
+			`fstat() = 0`,
+		`open(1,  <unfinished ...>` + "\n" +
+			`<... open resumed> , FLAG|FLAG) = -1 FLAG (sdfjfjfjf)` + "\n" +
+			`fstat() = 0`,
+		`open([USR1 IO], NULL, {tv_sec=5, tv_nsec=0}, 8 <unfinished ...>` + "\n" +
+			`<... rt_sigtimedwait resumed> )   = 10 (SIGUSR1)` + "\n" +
+			`fstat() = 0`,
+		`open(0, SNDCTL_TMR_START or TCSETS,` +
+			`{c_cc[VMIN]=1, c_cc[VTIME]=0} <unfinished ...>` + "\n" +
```

Comment:
These:
```
` + "\n" +
```
look a bit clumsy. Raw string literals support new lines, so I would something like this:
```
		`
open(0, SNDCTL_TMR_START or TCSETS, {c_cc[VMIN]=1, c_cc[VTIME]=0} <unfinished ...>
<... open resumed> , FLAG|FLAG) = -1 FLAG (sdfjfjfjf)
fstat() = 0
`,
```
This allows to read and write the input as is, without additional escaping and manually added new lines.

Reply:
Wow i overthought that. Will fix!